cloud/azure/serverfarms/detectors-azure-serverfarms.tf

```diff
@@ -0,0 +1,77 @@
+resource "signalfx_detector" "heartbeat" {
```

Comment:
Add `threshold` function & use `lasting` function instead of aggregation function for App Service Plan detectors.